extensions/webdav/src/test/java/org/exist/webdav/SerializationTest.java

```diff
@@ -39,15 +39,14 @@
 import java.io.IOException;
 import java.nio.file.Files;
 
-import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.junit.Assert.*;
 import static org.junit.Assert.assertEquals;
 
 public class SerializationTest {
 
     private static final String XML_WITH_DOCTYPE =
-            "<!DOCTYPE bookmap PUBLIC \"-//OASIS//DTD DITA BookMap//EN\" \"bookmap.dtd\">\n" +
-            "<bookmap id=\"bookmap-1\"/>";
+            "<!DOCTYPE bookmap PUBLIC \"-//OASIS//DTD DITA BookMap//EN\" \"bookmap.dtd\">" +
+            "<bookmap id=\"bookmap-1\"><title>The Title</title></bookmap>";
```

Comment:
Why add a child element here, it doesn't seem that it has changed anything else about the test?

Reply:
The child element was added to see the effect of `indent=no` to slightly more complex data.